HLTrigger/JetMET/interface/HLTJetSortedVBFFilter.h

```diff
@@ -36,6 +36,9 @@ class HLTJetSortedVBFFilter : public HLTFilter {
   static bool comparator ( const Jpair& l, const Jpair& r) {
     return l.first < r.first;
   }
+  static bool comparator_inv ( const Jpair& l, const Jpair& r) {
+    return l.first > r.first;
+  }
```

Comment:
comparator_inv is not uswed anywhere, so please remove these lines.
And if you want it in 73X (ie, before HLT moves to 74X), remember to make a 73X PR as well.